spec/helpers/users_helper_spec.rb

```diff
@@ -1,15 +1,15 @@
-require 'rails_helper'
```

Comment:
In general it's safe to delete a file you don't use, such as these helper test files.